spec/policies/planning_application_policy_spec.rb

```diff
@@ -22,23 +21,26 @@
 
   describe "#permitted_statuses" do
     context "an assessor" do
-      let(:user) { users(:assessor) }
+      let(:user) {  create :user, :assessor }
+      let(:policy) { described_class.new(user, record) }
```

Comment:
Why did we have to move the policy inside each context? I tried putting it back to where it was, and everything still passed for me. Maybe it was something you tried, but didn't undo?

Reply:
it had to be moved inside context because the block above iterates through the users then creates blocks for them. It will work for you if you copy paste this specific user and move it outside the context because then that user will be acessible to all the contexts. If you try removing the policy from the block just above this one youll see it break